README.md

```diff
@@ -306,10 +290,8 @@ will load the file into the interactive browser where it can be sorted and queri
 
 ## Updating / Installing dependencies
 
-We make use of python pip's support for only installing packages if their sha-256 hash matches a known good value.
-To add a new dependency, try installing the `hashin` python package that calculates a hash and adds it to the
-local `requirements.txt`. When installing the dependencies, always use `--require-hashes` to force this check
-on downloaded packages.
+We make use of Python pip's support for only installing packages if their sha-256 hash matches a known good value.
+To add a new dependency, use `pipenv install [package-name]`, which not only installs the package but Pipenv will also go to the trouble of updating the Pipfile as well.
```

Comment:
It might be worth mentioning here that both the Pipfile and Pipfile.lock will need to be committed to source control.